composer/models/bert/bert_hparams.py

```diff
@@ -50,8 +50,10 @@ def initialize_object(self) -> "BERTModel":
             model = transformers.AutoModelForSequenceClassification.from_pretrained(self.pretrained_model_name,
                                                                                     **model_hparams)
         else:
+            for k, v in model_hparams.items():
+                assert getattr(config, k) == v
```

Comment:
is this assert for validation or an invariant? if it's validation, can you make it into a ValueError?

Reply:
It's an invariant! The purpose is to ensure that internally casting from the model hparams to an HF config doesn't lose keys. I can add a comment describing why

Reply:
Discussed offline, but thoughts about hardcoding the model_hparams in line 55 and 57, since the dict contains one item with a constant key?